pepper-apis/src/appengine/app.yaml

```diff
@@ -3,16 +3,14 @@ runtime: java11
 instance_class: F1
 # look at tuning for spawning new VMs under load (10 connections -> new instance)
 
-# set 2 min instances
-
 manual_scaling:
   instances: 2
 
 network:
   instance_tag: study-server # necessary for firewall rules so that DSM will accept https requests from pepper
 
-# get size from env var, try this without mx, ms
-entrypoint: java -Dddp.firecloudKeysDir=fc_keys -Dconfig.file=application.conf DataDonationPlatform.jar
+# will need to move all contents of tcell into deploy, and copy tcell_agent.config after rending
+entrypoint: java -javaagent:tcell/tcellagent.jar-Dddp.firecloudKeysDir=fc_keys -Dconfig.file=application.conf DataDonationPlatform.jar
```

Comment:
Is it just me, or does checking in jars and dynamic libs into source seems a bit odd? Is there no way to manage this, say, through maven? Can we this into docker image builds?

Reply:
It is unusual, but I haven't found a supported mvn repo.   I like the idea to push it into docker image for builds and keep it out of "source".

Reply:
However...I'm not sure if this plays nice with GAE.  Maybe we could put the jars into buckets, then grab those as part of deploy?   CC @MocanaAtBroad

Reply:
After a bit of fiddling, I'm going to try to put the tcell tarball into a bucket (sans secrets) and then fetch that in circle.  Should be reproducible as long as we version that tarball accordingly and don't update it.

Reply:
🎉 woohoo!

Reply:
What I have done in the past with libraries that are not in Maven is to install them into local repo and then refer to them in Maven build as any other library.
Trick was to setup a module under main pom that when it got built it would grab the library and executed a maven install on it.
The solution here appears to work and this is just a runtime dependency, so maybe we leave it alone?